docs/source/index.md

```diff
@@ -28,25 +28,73 @@ The `truststore` module has a single API: `truststore.SSLContext`
    ctx = truststore.SSLContext()
 ```
 
+### Using truststore with pip
+
+Pip v22.2 released with experimental support for verifying certificates with system trust stores with truststore. To enable the feature use the flag `--use-feature=truststore` when installing a package like so:
+
+```{code-block} bash
+   # Install Django using system trust stores
+   $ python -m pip install --use-feature=truststore Django
+```
+
+This requires 'truststore' to be installed in the same environment as the one running pip and to be running Python 3.10 or later. For more information you can read the pip documentation about the feature.
```

Comment:
```suggestion
This requires `truststore` to be installed in the same environment as the one running pip and to be running Python 3.10 or later. For more information you can read the pip documentation about the feature.
```

Should we link to https://pip.pypa.io/en/stable/user_guide/#using-system-trust-stores-for-verifying-https ?

Reply:
Oh right, forgot to put a link because of https://github.com/pypa/pip/pull/11322 so wasn't sure if we should wait or not.

Reply:
Looks like there's a redirect being put in for the heading so we should be good to use it anyways. We can update once that PR gets merged.